build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
     id 'jacoco'
     id 'com.github.kt3k.coveralls' version '2.8.1'
     id 'application'
-    id 'checkstyle'
+    // id 'checkstyle'
```

Comment:
Style checks are broken probably because I/we have a version that is older than is expected in Gradle 5.0?